examples/hello-world/package.json

```diff
@@ -16,6 +21,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build"
+    "build": "react-scripts build",
+    "translate": "cross-env NODE_ENV=production babel ./src > NUL"
```

Comment:
Sorry for delay! I meant the created `NUL` file shouldn't be. I prefer non-redirecting to the `NUL` like this:
```
    "translate": "cross-env NODE_ENV=production babel ./src"
```